py2many/cli.py

```diff
@@ -131,11 +140,40 @@ def main():
             outdir = source.parent
         else:
             outdir = pathlib.Path(args.outdir)
-        print(f"Writing to: {outdir}")
-        output_path = outdir / (source.stem + settings.ext)
-        print(f"{filename}...{output_path}")
-        with open(output_path, "w") as f:
-            source_data = open(source).read()
-            f.write(transpile(source_data, settings.transpiler))
-        os.system(f"{settings.formatter} {output_path}")
-        print()
+        if os.path.isfile(filename):
```

Comment:
```
source.is_file()
```

Some of the code below reads a lot nicer with pathlib:

https://realpython.com/python-pathlib/

Reply:
done